pkg/skaffold/tag/tagger_mux.go

```diff
@@ -41,18 +41,16 @@ func (t *TaggerMux) GenerateTag(ctx context.Context, image latest.Artifact) (str
 func NewTaggerMux(runCtx *runcontext.RunContext) (Tagger, error) {
 	pipelines := runCtx.GetPipelines()
 	m := make(map[string]Tagger)
-	sl := make([]Tagger, len(pipelines))
 	for _, p := range pipelines {
 		t, err := getTagger(runCtx, &p.Build.TagPolicy)
 		if err != nil {
 			return nil, fmt.Errorf("creating tagger: %w", err)
 		}
-		sl = append(sl, t)
 		for _, a := range p.Build.Artifacts {
 			m[a.ImageName] = t
 		}
 	}
-	return &TaggerMux{taggers: sl, byImageName: m}, nil
+	return &TaggerMux{byImageName: m}, nil
```

Comment:
Can you also remove the field from the TaggerMux struct (on line 29)?

Thanks!

Reply:
My pleasure.

Please review again.